verifiers/internal/gha/npm_sigstore_tuf.go

```diff
@@ -0,0 +1,103 @@
+package gha
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	sigstoreTuf "github.com/sigstore/sigstore-go/pkg/tuf"
+)
+
+const (
+	attestationKeyUsage = "npm:attestations"
+	attestationKeyID    = "SHA256:jl3bwswu80PjjokCgh0o2w5c2U4LhQAE57gj9cz1kzA"
```

Comment:
is the attestationKeyID needed? Is that not the same as the pub key hint? I don't think the variable is used in the code

Reply:
thanks, removed.